app/src/main/java/org/greenstand/android/TreeTracker/utilities/LocationDataConfig.kt

```diff
@@ -0,0 +1,17 @@
+package org.greenstand.android.TreeTracker.utilities
+
+/**
+ *  app-wide constants and utilities
+ */
+object LocationDataConfig {

```

Comment:
Eventually this would be refactored when UI configuration for controlling convergence parameter is built